controls/cis_rhel8.yml

```diff
@@ -977,7 +977,8 @@ controls:
     notes: >-
       The rule also configures correct run level to prevent unbootable system.
     rules:
-      - xwindows_remove_packages
+      - package_xorg-x11-server-common_removed
```

Comment:
I think we should remove the cis@rhel7 reference from xwindows_remove_packages.

Reply:
I will address RHEL 7 in a separate PR. Both approaches work there, I have just tested that. Either using xwindows_remove_packages or package_xorg-x11-server-common_removed together with xwindows_runlevel_target. The latter is more aligned with the policy.

Reply:
Oh, I meant cis@rhel8. Sorry. If I am not wrong, there are some packages which depends on `xorg-x11-server-utils` and this requirement were breaking valid scenarios, so the CIS policy was updated to not remove it after a discussion about this issue. Removing only the `xorg-x11-server-common` should be enough for this requirement.

Reply:
fixed

Reply:
Great. Thanks.